liblouis/compileTranslationTable.c

```diff
@@ -3479,10 +3480,18 @@ compileRule(FileInfo *nested, TranslationTableOffset *newRuleOffset,
 			if (ok && newRuleOffset) *newRuleOffset = ruleOffset;
 			break;
 		}
+		case CTO_NoCross:
+			if (nocross) {
+				compileError(nested, "%s already specified.",
+						_lou_findOpcodeName(CTO_NoCross));
+				ok = 0;
+				break;
+			}
+			nocross = 1;
+			goto doOpcode;
 		case CTO_Syllable:
 			(*table)->syllables = 1;
 		case CTO_Always:
-		case CTO_NoCross:
```

Comment:
You remove the `case CTO_NoCross` from the switch statements. Should we not also remove it from the enum?

Reply:
Hmm, maybe, not sure if we still need it elsewhere?

Reply:
Actually I move it to a few lines above.